.github/workflows/pr-push.yml

```diff
@@ -0,0 +1,32 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - gh-pages
+  pull_request: {}
+
+jobs:
+  ci:
+    name: Validate and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10
```

Comment:
There is no LTS still, right?

Reply:
Nope. Only numbers are supported presently

Reply:
Maybe we should use v12 now, as it'll be LTS within a month.

Reply:
sounds like a plan!